tests-config/setup.js

```diff
@@ -1,52 +1,71 @@
-'use strict'
+"use strict";
 
-const fs = require('fs')
-const { extname } = require('path')
-const unified = require('unified')
-const parse = require('remark-parse')
-const stringify = require('remark-stringify')
-const frontmatter = require('remark-frontmatter')
-const footnotes = require('remark-footnotes')
-const pangu = require('../index')
+const fs = require("fs");
+const { extname } = require("path");
+const unified = require("unified");
+const parse = require("remark-parse");
+const stringify = require("remark-stringify");
+const frontmatter = require("remark-frontmatter");
+const footnotes = require("remark-footnotes");
+const pangu = require("../index");
 
-const processor = unified()
-  .use(parse, { footnotes: true })
-  .use(stringify)
-  .use(frontmatter)
-  .use(footnotes, {inlineNotes: true})
-  .use(pangu)
-  .freeze()
+const processor = function (options) {
```

Comment:
这个叫 `createProcessor` 更准确一些：

```suggestion
const createProcessor = function (options) {
```